routes/web.php

```diff
@@ -12,7 +12,5 @@
 */
 
 Route::get('/', function () {
```

Comment:
I guess. It's just the dashboard, so I thought it would be easiest for Kat and Steph to find that way. It's not a www domain anyway. arcvservice-prealpha is pretty obscure to me.